packages/lmi/tests/test_rpm.py

```diff
@@ -0,0 +1,131 @@
+import asyncio
+import time
+from typing import List
+import pytest
+from lmi.llms import LiteLLMModel
+from lmi.types import Message
+from limits import RateLimitItemPerMinute
+from lmi.rate_limiter import GLOBAL_LIMITER 
+
+
+async def test_rpm_limit():
+    """Test the request rate limit (RPM) functionality"""
+    messages = [Message(role="user", content="Hi")]
+    results = {}
+    
+    # Test request time without rate limit
+    print("\n=== Testing without rate limit ===")
+    model_no_limit = LiteLLMModel(
+        name="deepseek/deepseek-chat",
+        config={
+            "model_list": [{
+                "model_name": "deepseek/deepseek-chat",
+                "litellm_params": {
+                    "model": "deepseek/deepseek-chat",
+                    "temperature": 0.1,
+                    "max_tokens": 4096,
+                }
+            }]
+        }
+    )
+    
+    results["No Limit"] = {}
+    for req_count in [2, 3, 4]:
+        print(f"\nTesting time to complete {req_count} requests without limit...")
+        start_time = time.time()
```

Comment:
Let's use `time.perf_counter` to be more precise, on Windows `time.sleep` can be pretty coarse